src/utils/Utils.ts

```diff
@@ -1556,4 +1560,29 @@ export default class Utils {
   public static unescapeCsvValue(value: any): void {
     // double quotes are handle by csvToJson
   }
+
+  public static sanitizeCSVExport(data: any): any {
+    if (!data || typeof data === 'number' || typeof data === 'bigint' || typeof data === 'symbol' || Utils.isBoolean(data) || typeof data === 'function') {
+      return data;
+    } else if (typeof data === 'string') {
```

Comment:
[low] You can chain with simple 'if' statements, no need the 'else if', just to make it more readable.